src/elements/content-sidebar/activity-feed/task-new/flowTypes.js

```diff
@@ -0,0 +1,96 @@
+// @flow
+import type { FileMini, FolderMini, ISODate, UserMini } from '../../../../common/box-types';
+
+type ID = string;
+
+export type TaskCollabStatus = 'APPROVED' | 'REJECTED' | 'COMPLETED' | 'NOT_STARTED';
+
+export type TaskStatus = TaskCollabStatus | 'IN_PROGRESS';
+
+type TaskMini = {|
+    created_at: ISODate,
+    id: ID,
+    modified_at: ISODate,
+    status: TaskStatus,
+    type: 'task',
+|};
+
+type TaskCollab<R> = {|
+    completed_at?: ?ISODate,
+    id: ID,
+    role: R,
+    status: TaskCollabStatus,
+    target: UserMini,
+    task?: TaskMini,
+    type: 'task_collaborator',
+|};
+
+type TaskCollabCreator = TaskCollab<'CREATOR'>;
+
+export type TaskCollabAssignee = {|
+    ...TaskCollab<'ASSIGNEE'>,
+    permissions: {|
+        can_delete: boolean,
+        can_update: boolean,
+    |},
+|};
+
+export type TaskLink = {|
```

Comment:
I would move all exports to the bottom like we do for other files

Reply:
Done